companies/123greetings.json

```diff
@@ -1,22 +1,24 @@
 {
     "slug": "123greetings",
     "relevant-countries": [
-        "all"
+        "all",
+        "dong"
     ],
     "categories": [
         "entertainment"
     ],
     "name": "123Greetings.com Inc.",
-    "address": "255 Executive Drive, Suite 400\nPlainview New York 11803\nUnited States of America",
+    "address": "123Greetings.com Inc.\n255 Executive Drive, Suite 400\nPlainview New York 11803\nUnited States of America",
     "fax": "+1 212 202 4738",
     "email": "privacy@123greetings-inc.com",
-    "web": "https://www.123greetings.com",
+    "web": "%https://www.123greetings.com",
```

Comment:
🚫 **[companylint]** <[nsfw-web](https://github.com/datenanfragen/data/issues/1369)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Record has `nsfw` set to `true` but also specifies `web`.

If this company is really not safe for work, remove the website.
```suggestion
```